engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/MetaKindNode.java

```diff
@@ -0,0 +1,38 @@
+package org.enso.interpreter.node.expression.builtin.meta;
+
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.AcceptsError;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.enso.interpreter.runtime.callable.UnresolvedConversion;
+import org.enso.interpreter.runtime.callable.UnresolvedSymbol;
+import org.enso.interpreter.runtime.data.Type;
+import org.enso.interpreter.runtime.data.atom.Atom;
+import org.enso.interpreter.runtime.error.DataflowError;
+
+@BuiltinMethod(
+    type = "Meta",
+    name = "get_kind_builtin",
+    description = "Finds kind of a value",
+    autoRegister = false)
+final class MetaKindNode extends Node {
+  final long execute(@AcceptsError Object value) {
+    if (value instanceof Atom) {
+      return 2;
```

Comment:
I expected _magic constant_ complains...

Reply:
Fair. It is indeed well-tested I think.

I'd just add a comment in both files mentioning that we want to be consistent with the other file - so if any modification has to happen it is easier to find where the relevant parts are.

Reply:
I don't want to risk loosing the green CI status.